packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -76,6 +76,10 @@ export default function ButtonAppBar({
                 title: "אודות",
                 onClick: () => history.push("/" + MainNavigationKeys.About),
               },
+              {
+                title: "צור קשר",
+                onClick: () => history.push("/" + MainNavigationKeys.Contact),
```

Comment:
Until icons are added, let's not add this page to the menu

Reply:
Done